docs/api/nav.md

```diff
@@ -50,15 +35,17 @@ import NavLinkExample from '@site/static/usage/nav/nav-link/index.md';
 
 ## Navigation within a Modal
 
-TODO: Playground Example
+Modal can use Nav to offer a linear navigation that is independent of the URL.
+
+:::note
 
-## Animations
+The example below uses a reference to Nav and the public method APIs to push and pop views. It is recommended to use NavLink in implementations that do not require this level of granular access and control.
 
-TODO: Playground Example
+:::
 
-## Event Handling
+import ModalNavigationExample from '@site/static/usage/nav/modal-navigation/index.md';
 
-TODO: Playground Example
+<ModalNavigationExample />
```

Comment:
This is a strange one... the style block that is causing this issue is:
```css
ion-modal .ion-page:not(ion-nav.ion-page) {
    position: relative;
    contain: layout style;
    height: 100%;
}
```

somehow it's stripping the space within the `:not` tag, as ours is:
```css
ion-modal .ion-page:not(ion-nav .ion-page) {
    position: relative;
    contain: layout style;
    height: 100%;
}
```

We can see that issue directly from the CDN: https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css.

I'll need to work with Liam to uncover why this is happening. I don't see this as a blocker though, as the Stackblitz examples work & when developers install the package they will not have this issue.

Reply:
Ok, this will be resolved once: https://github.com/ionic-team/ionic-framework/pull/25767 is merged, we deploy the next patch & jsDelivr cache is purged.

Reply:
Word 👍